app/app.go

```diff
@@ -372,6 +372,11 @@ func SetUpgradeConfig(upgradeConfig *config.UpgradeConfig) {
 		bridge.TransferOutMsg{}.Type(),
 		oracle.ClaimMsg{}.Type(),
 	)
+	upgrade.Mgr.RegisterMsgTypes(upgrade.BEPHHH,
+		stake.MsgEditValidator{}.Type(),
+		stake.MsgRedelegate{}.Type(),
```

Comment:
Do we need register `delegate` transaction type?

Reply:
yes, we do. Added